examples/lusid/instruments/query-instruments.sql

```diff
@@ -0,0 +1,25 @@
+-- ============================================================
+
+-- In this query, we will filter various types of instruments.
+-- These can be uploaded via the other examples in this folder.
+-- We will query three different types of instrument,
+-- and join the results using union.
+
+-- ============================================================
+
+-- Select common columns and filter
+select DisplayName, ClientInternal, DomCcy, LusidInstrumentId
+from Lusid.Instrument.Equity
+where DisplayName = 'Admiral Group'
```

Comment:
We'd usually filter on an Instrument ID which are more likely to be unique.

Reply:
Changes made.